primitives/identifier/src/lib.rs

```diff
@@ -118,3 +118,13 @@ impl<T: Config> IdentifierUpdate<IdentifierOf, IdentifierTypeOf, EventEntryOf, I
 		.map_err(|_| IdentifierError::MaxEventsHistoryExceeded) // Map DispatchError to your custom Error
 	}
 }
+
+impl<T: Config> Pallet<T> {
+	pub fn identifier_exists(id: &IdentifierOf) -> bool {
+		Identifiers::<T>::iter_prefix(id).next().is_some()
+	}
+
+	pub fn identifier_exists_by_type(id: &IdentifierOf, id_type: IdentifierTypeOf) -> bool {
```

Comment:
Can the type be optional? (if the above call is not returning content).

Reply:
For unknown type `identifiers_exist` can be used. `identifier_exists_by_type` is faster when which pallet type is known.